src/components/ColorPicker/docs/examples.ts

```diff
@@ -0,0 +1,12 @@
+export const importExample = `import { ColorPicker } from 'wix-ui-tpa/ColorPicker';`;
+
+export const example = `
+<ColorPicker>
+  <ColorPicker.Item ariaLabel="first color" color="red" />
+  <ColorPicker.Item ariaLabel="second color" color="green" />
+  <ColorPicker.Item ariaLabel="third color" color="yellow" />
+  <ColorPicker.Item ariaLabel="fourth color" color="brown" />
+  <ColorPicker.Item ariaLabel="fifth color" color="#2318FF" />
+  <ColorPicker.Item ariaLabel="sixth color" />
```

Comment:
I think it's missing a color here, no?

Reply:
@jonathanadlerI made a fallback color, which is white. not sure if we should keep it or not

Reply:
If you have a fallback value then `ColorPickItemProps.color` should be optional (currently is it mandatory)

Reply:
removed fallback color